test/plugin/scenarios/lettuce-scenario/support-version.list

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+5.1.3.RELEASE
```

Comment:
Latest version is 5.2.1.RELEASE.

Reply:
OK, I will update after test.

Reply:
Check the execution time, if it is longer than 35 mins in every group, consider to add a new one.

Reply:
OK, has out 35 mins after add all version, so add a new group.